include/sys/queue.h

```diff
@@ -0,0 +1,832 @@
+/****************************************************************************
+ * include/sys/queue.h
```

Comment:
Should this be controlled by `CONFIG_ALLOW_BSD_COMPONENTS`?

Reply:
Also need to add it to LICENSE.